core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -203,14 +204,21 @@ var TopicEditorPage = function() {
   };
 
   this.changeSubtopicPageContents = async function(richTextInstructions) {
-    var subtopicPageEditor = await forms.RichTextEditor(
-      pageEditor);
-    await subtopicPageEditor.clear();
-    await richTextInstructions(subtopicPageEditor);
+    var subtopicPageContentButton = element(by.css(
+      '.protractor-test-edit-html-content'));
+    await waitFor.elementToBeClickable(subtopicPageContentButton,
+      'Edit subtopic htm content button taking too long to be clickable');
+    await subtopicPageContentButton.click();
+    var pageEditor = element(by.css(
+      '.protractor-test-edit-subtopic-page-contents'));
+    await waitFor.visibilityOf(pageEditor,
+      'Subtopic html editor takes too long to appear');
+    await (
+      await browser.switchTo().activeElement()).sendKeys(richTextInstructions);
```

Comment:
Did you change what `richTextInstructions` is supposed to be? It used to be a function, not a string